src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -856,9 +905,14 @@ public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
 
                     await server.AcceptConnectionAsync(async connection =>
                     {
-                        await connection.ReadRequestHeaderAndSendCustomResponseAsync(
+                        HttpRequestData requestData = await connection.ReadRequestDataAsync();
+#if TARGET_BROWSER
```

Comment:
Wouldn't this be worth putting in `ReadRequestHeaderAndSendCustomResponseAsync` directly as with `ReadRequestHeaderAndSendResponseAsync`?

Reply:
To recognize pre-flight request I we `HttpRequestData` rather than `List<string> lines` which `ReadRequestHeaderAndSendCustomResponseAsync` has. And `ReadRequestHeaderAndSendCustomResponseAsync` is used in lot of places I didn't want to touch.